be/src/runtime/mem_tracker.h

```diff
@@ -170,6 +172,12 @@ class MemTracker : public std::enable_shared_from_this<MemTracker> {
             Release(-bytes);
             return true;
         }
+        if (MemInfo::current_mem() + bytes >= MemInfo::mem_limit()) {
+            VLOG_RPC << "TryConsume failed, bytes=" << bytes
+                     << " process whole consumption=" << MemInfo::current_mem()
+                     << " mem limit=" << limit << " attempting to GC";
```

Comment:
That's may logging not accurate memory usage for that moment if set `refresh_current_memory_usage_interval_ms` longer? we experience memory usage increasing fast in seconds.

Reply:
get mem usage may an expensive call